homeassistant/components/openai_conversation/__init__.py

```diff
@@ -52,17 +56,38 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     async def render_image(call: ServiceCall) -> ServiceResponse:
         """Render an image with dall-e."""
+        client = hass.data[DOMAIN][call.data["config_entry"]]
+
+        if call.data["size"] in ("256", "512", "1024"):
+            ir.async_create_issue(
+                hass,
+                DOMAIN,
+                "image_size_deprecated_format",
+                breaks_in_ha_version="2024.7.0",
+                is_fixable=False,
+                is_persistent=True,
+                learn_more_url="https://platform.openai.com/docs/api-reference/images/create",
```

Comment:
Shouldn't we link to our openai docs instead ?

Reply:
You are correct, thank you!
Done.